src/controllers/commands/CommandController.cpp

```diff
@@ -203,6 +206,54 @@ QString CommandController::execCommand(const QString &text, ChannelPtr channel,
                     channel->addMessage(Message::createSystemMessage(message));
                 });
 
+                return "";
+            } else if (commandName == "/follow" && words.size() >= 2) {
+                auto app = getApp();
+
+                auto user = app->accounts->twitch.getCurrent();
+                auto target = words.at(1);
+                if (user->isAnon()) {
+                    channel->addMessage(
+                        Message::createSystemMessage("You must be logged in to follow someone"));
+

```

Comment:
remove empty line

Reply:
I would not say this line is empty

Reply:
Comparing to all other isAnon checks in `CommandController.cpp`:
https://github.com/apa420/chatterino2/blob/c26e18c2bf8adc0aa0c636aee25db0f652c545ce/src/controllers/commands/CommandController.cpp#L182
https://github.com/apa420/chatterino2/blob/c26e18c2bf8adc0aa0c636aee25db0f652c545ce/src/controllers/commands/CommandController.cpp#L199
Including the one in /unfollow
https://github.com/apa420/chatterino2/blob/c26e18c2bf8adc0aa0c636aee25db0f652c545ce/src/controllers/commands/CommandController.cpp#L239
it just doesn't fit

Reply:
oh sorry, misinterpreted what you said, I see what you mean now